spec/features/user/user_signup_spec.rb

```diff
@@ -19,21 +19,6 @@
     expect(current_path).to eq(user_path(User.find(1)))
   end
 
-  xit "can't sign up with an invalid email" do
-    within(".navbar-nav") do
-      click_link_or_button("Sign up")
-    end
-    within("#signUpModal") do
-      fill_in("name", with: "Paul")
-      fill_in("email", with: "hi")
-      fill_in("password", with: "test")
-      click_link_or_button("Sign up")
-    end
-    expect(User.count).to eq(0)
-    expect(page).to have_content("Input is invalid, please try again")
-    expect(current_path).to eq(users_path)
-  end
-
```

Comment:
Why are we no longer validating the email field? I don't think we should just blow over this test. I think we should use a `validation unless` clause. Something like:
```ruby
validates :email,
    format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i },
    if: email_login?(user)
```

Or something. Not entire sure how this would work, but I think we can make it happen. I'm down to work on it depending how you feel.